GHzDACs/ghz_fpga_server.py

```diff
@@ -1855,8 +1963,46 @@ def dac_bist(self, c, chan, data):
         cmd, shift = dac.DAC.getCommand({'A': (2, 0), 'B': (3, 14)}, chan)
         dev = self.selectedDAC(c)
         ans = yield dev.runBIST(cmd, shift, data)
-        returnValue(ans)
-    
+        # This is coming back with 64-bit ints, the coercing of which needs to be fixed in pylabrad
```

Comment:
Have you checked that this won't break Setting 1300 (DAC Bringup) or bringup for non-JT boards?